__tests__/app.test.js

```diff
@@ -465,6 +464,15 @@ describe("GET /api/reviews (queries)", () => {
         expect(msg).toBe("Invalid sort by option");
       });
   });
+  test("400: responds with error when given invalid order by option", () => {
+    return request(app)
+      .get("/api/reviews?order=sideways")
+      .expect(400)
+      .then((response) => {
+        const { msg } = response.body;
+        expect(msg).toBe("Invalid order option");
+      });
```

Comment:
Missing one happy path test: Your data set has a category which has no associated reviews (you'll need to check your data files to find this)
status 200: responds with empty articles array